frontend/.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
```

Comment:
Failed to commit under root folder with exception:

`npx: command not found`

Reply:
I may use npm scripts directly without npx.

Reply:
Fixed. @ecmadao PTAL

Reply:
Tried in local, found another interesting issue. Seems after the husky hook installed, our existed pre-commit cannot work anymore, and the `pre-commit install` will throw exceptions:

```bash
[ERROR] Cowardly refusing to install hooks with `core.hooksPath` set.
hint: `git config --unset-all core.hooksPath
```

Reply:
Merged your settings into this.
Now we can:
- [x] run `pre-commit install` correctly.
- [x] block committing when something goes wrong.
- [x] run eslint only on `frontend/**/*.{js,ts,jsx,tsx,vue}`, with `frontend/.eslintrc.js` loaded correctly.
PTAL